# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: D:\Documents and Settings\jherr\My Documents\NetBeansProjects\clearcase-hudson-plugin\src\main\java\hudson\plugins\clearcase
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: AbstractClearCaseScm.java
--- AbstractClearCaseScm.java Base (BASE)
+++ AbstractClearCaseScm.java Locally Modified (Based On LOCAL)
@@ -283,11 +283,16 @@
             if (normViewPath != null) {
                 return workspace.child(normViewPath);
             } else {
+                normViewPath = getViewPath();
+                if (normViewPath != null) {
+                    return workspace.child(normViewPath);
+                } else {
                 // Should never happen, because viewName must not be null, and if viewpath is null, then it is made equal to viewName
                 throw new IllegalStateException("View path name cannot be null. There is a bug inside AbstractClearCaseScm.");
             }
         }
     }
+    }
 
     public String getViewName() {
         return viewName;
